db/migrate/20190719132900_add_subdomain_in_companies.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubdomainInCompanies < ActiveRecord::Migration
+  def change
+    add_column :companies, :subdomain, :string
```

Comment:
add character limit validation in the company model
and add its test case as well

Reply:
limit already in the model as well. plus the tested cases for company are already there.

Reply:
I don't see them in this PR
can you verify? if you added them in this PR

Reply:
verified already there